tardis/utilities/utils.py

```diff
@@ -98,6 +99,15 @@ def machine_meta_data_translation(
         raise
 
 
+def str_to_state(resources):
+    for entry in resources:
+        state_class = getattr(
+            import_module(name="tardis.resources.dronestates"), f"{entry['state']}"
+        )
+        entry["state"] = state_class()
```

Comment:
I suggest to reduce the "magic" here a bit. I don't see a need to use `import_module` since the target is constant, and the f-string could just be `str` (provided it isn't a string to begin with).
```suggestion
        state_class = getattr(
            tardis.resources.dronestates, str(entry['state'])
        )
        entry["state"] = state_class()
```
This would need an `import tardis.resources.dronestates` either at the module scope or, if there are dependency issues, inside the function.